README.md

```diff
@@ -0,0 +1,65 @@
+# Celery plugin for [Tutor](https://docs.tutor.edly.io)
+
+A tutor plugin to extend the default LMS and CMS celery workers included in Tutor.
+It adds and configures extra deployments running LMS and CMS celery workers where
+every deployment will process async tasks routed to a specific queue. Having this
+workers separation per queue can help to define the scaling requirements for the Celery
+deployments, since having a single queue (the default one) with a single deployment can
+lead to unexpected behaviors when running large-scale sites.
+
+## Installation
+
+```shell
+pip install git+https://github.com/eduNEXT/tutor-contrib-celery
+```
+
+## Usage
+
+```shell
+tutor plugins enable celery
+```
+
+## Configuration
+
+### Celery queues
+
+By default, tutor-contrib-celery enables the following queues with independent deployments
+for each:
+
+- **CMS**: default, high, low (taken from CMS settings [here](https://github.com/openedx/edx-platform/blob/open-release/redwood.master/cms/envs/common.py#L1578-L1582))
+- **LMS**: default, high, high_mem (taken from LMS settings [here](https://github.com/openedx/edx-platform/blob/open-release/redwood.master/lms/envs/common.py#L2913-L2917))
+
+> [!NOTE]
+> We recommend using [tutor-contrib-pod-autoscaling](https://github.com/eduNEXT/tutor-contrib-pod-autoscaling)
+> to setup requested resources and limits.
+
+In case you are using different celery queues than the defaults from Open edX, you can
+extend the list by setting `CELERY_WORKER_VARIANTS` on your `config.yml`. The format is the following:
+
+```yaml
+CELERY_WORKER_VARIANTS:
+  lms:
+    - high
+    - high_mem
+    - lms_custom_queue
+  cms:
+    - high
+    - low
+    - cms_custom_queue
+```
+
+This plugin also provides a setting to directly route LMS/CMS tasks to an specific queue:
```

Comment:
I think it's important to mention that this will extend/override the variable `EXPLICIT_QUEUES` on both LMS and CMS